src/redux/actions/sync.js

```diff
@@ -142,6 +142,13 @@ export function doGetSync(passedPassword, callback) {
             },
           });
 
+          // Temp solution until we have a bad password error code
+          // Don't fail on blank passwords so we don't show a "password error" message
+          // before users have ever entered a password
+          if (password !== '') {
+            dispatch({ type: ACTIONS.SYNC_APPLY_BAD_PASSWORD });
+          }
+
```

Comment:
This seems quite ugly. It's still quite possible, although rare for `sync_apply` to fail for other reasons. I guess we could just assume every failure is a bad password failure so that the user can try again.

Reply:
https://github.com/lbryio/lbry-sdk/issues/2175

Reply:
Yeah I agree. I intend to change this once https://github.com/lbryio/lbry-sdk/issues/2175 is resolved.